lua/possession/paths.lua

```diff
@@ -34,6 +34,9 @@ end
 
 --- Vim expands the given dir, then converts it to an absolute path
 function M.absolute_dir(dir)
+    -- TODO: Strip trailing forward/back slash after calling absolute.
+    -- The cwd in the JSON has no trailing slash, so if we don't remove
+    -- it then the dir will not match that cwd in the JSON.
     return Path:new(vim.fn.expand(dir)):absolute()
```

Comment:
Basically, as per the code comment.

This is an outstanding issue here. When using the command autocomplete (at least for my autocomplete, which is nvim-cmp), folder names get a trailing slash, and this doesn't match the JSON cwd, which has no trailing slash.

I couldn't find a nice way to remove the trailing slash that would work across OS platforms. Plenary looks to have a clean function that does this but it's internal.

We could brute force it by gsub-ing for both / and \ at the end. But if you know a better way, let me know.

Reply:
In newer versions of Neovim there is `vim.fs.normalize` but I am not sure when it was introduced and maybe we can for now leave without it. I think we can use the following:
```lua
if vim.endswith(p, Path.path.sep) then
	p = p:sub(1, #p - 1)
end
```

Reply:
That code looks to have worked, thanks.